pymathics/graph/format.py

```diff
@@ -52,7 +52,7 @@ def format_graph(G, **options) -> str:
         # "edgecolors": "black",  # Set below
         # "width": 5,             # Marmonized
     }
-    draw_options.update(options)
+    # draw_options.update(options)
```

Comment:
What is the reason this should be commented out? Inquiring developers want to know!